go.mod

```diff
@@ -2,6 +2,11 @@ module github.com/RHEnVision/provisioning-backend
 
 go 1.18
 
+// go get https://github.com/georgysavva/scany/v2@v2.0.0-alpha.2
+// go get github.com/jackc/tern/migrate@v2.0.0-beta.1
+
+replace github.com/jackc/tern => github.com/lzap/tern2 v0.0.0-20220813151204-54e00e2a8f9d
+
```

Comment:
This will be removed once `tern` v2 is properly tagged, the two `go get` commands are required until stable versions are released tho.

I suggest to go ahead and merge this so we can start testing this on stage.

Reply:
hump, we will need to convince our linter tho.
Also do we have any timelines for the releases.
I'm fine being early adopter, but only if we know the releases are likely to happen in timely manner.